pkg/storaged/fsys-tab.jsx

```diff
@@ -243,18 +241,19 @@ export function mounting_dialog(client, block, mode, forced_options) {
             if (mode == "mount" || (mode == "update" && is_filesystem_mounted)) {
                 return (get_block_fsys()
                         .then(block_fsys => {
-                            return (block_fsys.Mount({ })
+                            const block = client.blocks[block_fsys.path];
+                            return (client.mount_at(block, new_dir)
                                     .catch(error => {
                                         // systemd might have mounted the filesystem for us after
                                         // unlocking, because fstab told it to.  Ignore any error
                                         // from mounting in that case.  This only happens when this
                                         // code runs to fix a inconsistent mount.
-                                        return (utils.is_mounted_synch(client.blocks[block_fsys.path])
+                                        return (utils.is_mounted_synch(block)
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-16991-20220825-111327-558828a7-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)